packages/lesswrong/server/scripts/lessOnlineMessaging.tsx

```diff
@@ -0,0 +1,142 @@
+      </li>
+      <li>
+        <b>There’ll be optional entertainment</b> including a weekend-long puzzle hunt throughout <b><a href="https://www.lighthaven.space/">Lighthaven</a></b> and also a rationalist dance party led by <b><a href="https://www.lesswrong.com/posts/YMo5PuXnZDwRjhHhE/lesswrong-s-first-album-i-have-been-a-good-bing">The Fooming Shoggoths</a></b> (with many unreleased tracks being played).
+      </li>
+      <li>
+        <b>And anything else that attendees want to do!</b> There’ll be a google sheet that anyone can edit to add activities or sessions.
+      </li>
+    </ul>
+    <p>
+      Personally, I started working on LessWrong in 2017, and I’d love to meet more of you people who’ve read and contributed to the blogosphere and the broader scene. There’s 25,000 people getting this email (everyone active on LessWrong since January 2020), and I believe that some of the most interesting and thoughtful and disagreeable and characterful people alive are on this mailing list, and I hope many of you come :-)
+
+    </p>
+    <p>
+      — Ben Pace & the <b><a href="https://less.online/">LessOnline</a></b> Team
+    </p>
+    <hr/>
+    <p>
+      P.S. This festival started as an idea when we were talking to the team behind the forecasting & prediction market festival <b><a href="https://www.manifest.is/">Manifest</a></b>. They ran a smash-hit event last year at Lighthaven, and this year they encouraged us to run our own event as part of a series. This means that the weekend right after LessOnline is Manifest, and we've <b><a href="https://less.online/#tickets-section">teamed up to offer a discounted ticket to both</a></b>. Already many people have made plans to come for both and will be staying at Lighthaven for the week in between!
+    </p>
+    <p>
+      P.P.S. <b>Got questions?</b> You can’t reply to this email, but <b>comments and questions are welcome in <a href="https://www.lesswrong.com/posts/MmWziepD8DDauSide/lessonline-festival-updates-thread">this LW thread</a></b> and/or using <b>the intercom bubble</b> in the bottom right of the <a href="https://less.online/">less.online</a> or <a href="http://lesswrong.com">lesswrong.com</a> websites.
+
+    </p>
+    <img src="https://res.cloudinary.com/lesswrong-2-0/image/upload/v1712166847/habryka_6_g84ubl.png"/>
+  </div>
+}
+
+const lessOnlineMessaging = async () => {
+  // const lwevents = await LWEvents.find({createdAt: {$gte: new Date("2024-01-28")}}).fetch()
+  // console.log(lwevents.length)
+
+  // console.log(lwevents.length, Array.from(new Set(lwevents.map(event => event.userId))).length)
+
+  // const users = await Users.find({
+  //   lastNotificationsCheck: {$gt: new Date("2020-01-01")},
+  //   $or: [{ banned: { $exists: false } }, { banned: { $lte: new Date() } }],
+  //   karma: {$gte: 0}
```

Comment:
don't forget `deleted: false`, and then also `{ sort: { createdAt: 1 } }` in the options